app/org/sagebionetworks/bridge/services/ExportService.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.bridge.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+
+/** Service that interfaces Bridge Server with Bridge Exporter. */
+public interface ExportService {
+    /** Kicks off an on-demand export for the given study. */
+    void startOnDemandExport(StudyIdentifier studyId) throws JsonProcessingException;
+}
```

Comment:
I can see it both ways.

On the one hand, this has a direct dependency on SQS, which most of the other services don't. An interface helps us keep SQS-specific details from leaking into other classes. (See also SendEmailService, UserDataDownloadService.)

On the other hand, this service encapsulates a very simple SQS call that's very easy to replace. Contrast with the DAOs, which have a lot of complexity in them around mapping data types to DDB types and CRUDs.